packages/core/src/transports/initialize.ts

```diff
@@ -102,37 +102,67 @@ export function initializeTransports(
     return filteredItems;
   };
 
-  const send = (items: TransportItem[]) => {
+  const batchedSend = (items: TransportItem[]) => {
```

Comment:
Suggestion:
Maybe we can rename this to better reflect that thsi send method if only for the BatchExecutor?
E. g. `batchExecuterSend`